DependencyInjection/Configuration.php

```diff
@@ -50,6 +50,12 @@ public function getConfigTreeBuilder()
                         ->scalarNode('password')->end()
                     ->end()
                 ->end()
+                ->arrayNode('html5validation')
+                    ->children()
+                        ->scalarNode('url')->end()
+                        ->arrayNode('ignores_extract')->end()
```

Comment:
Should the `liip_functional_test.html5validation.ignores` and `liip_functional_test.html5validation.ignores_extract` parameter be kept? It looks like it's used to ignore validation errors. These lines haven't been updated since 2011. Should I open an issue to talk about removing unused features?